### ISSUE TITLE
Allow the WebID and OIDC providers to reside on different domains

### ISSUE THREAD

**Issue by RubenVerborgh** [solid/node-solid-server#521]
Right now, OIDC providers can only authenticate users on their own domain.

However, the WebID resource could explicitly indicated trusted OIDC domains or providers. For example, then I'd be able to authenticate as https://ruben.verborgh.org/profile/#me from https://drive.verborgh.org/ with OIDC. (Right now, I can only do that via WebID+TLS.)

**Comment by elf-pavlik** [comment]
Searching for 'OpenID Connect delegation', I've found this StackOverflow answer https://stackoverflow.com/a/28022632 which links to http://openid.net/specs/openid-connect-discovery-1_0.html#URLSyntax

Would something like below work?

```turtle
https://ruben.verborgh.org/profile/#me
  http://openid.net/specs/connect/1.0/issuer https://drive.verborgh.org/ .
```

**Comment by dmitrizagidulin** [comment]
great find, @elf-pavlik , re `http://openid.net/specs/connect/1.0/issuer` rel!

**Comment by dmitrizagidulin** [comment]
@elf-pavlik After some discussion with @timbl, we decided not to go with `http://openid.net/specs/connect/1.0/issuer`, use `solid:oidcIssuer` instead:

```ttl
<https://ruben.verborgh.org/profile/#me>
  <http://www.w3.org/ns/solid/terms#oidcIssuer> <https://drive.verborgh.org> .
```

**Comment by dmitrizagidulin** [comment]
I suppose we can add some sort of rdfs/owl equivalence to the specs issuer link, but I'm not familiar enough with ontologies to do that. Feel free to make an addition to https://github.com/solid/vocab/pull/23, though.

**Comment by RubenVerborgh** [comment]
Depends on the semantics. Are they equivalent? Or is `solid:oidcIssuer` something more specific?

**Comment by dmitrizagidulin** [comment]
You're right, I suppose `solid:oidcIssuer` is more specific; it denotes that the server supports the WebID-OIDC protocol specifically.

**Comment by RubenVerborgh** [comment]
Done: https://github.com/solid/vocab/pull/23/commits/c70a6c4e3a9171a7f3ced046b5bed7bfcf5ec166

**Comment by dmitrizagidulin** [comment]
Nice!

**Comment by RubenVerborgh** [comment]
Fixed by #545.

**Comment by RubenVerborgh** [comment]
Ah no, wait, #545 only covers WebID-TLS. This issue requests the same functionality as #545, but with OIDC.

**Comment by dmitrizagidulin** [comment]
Should be addressed by PR https://github.com/solid/node-solid-server/pull/553

**Comment by dmitrizagidulin** [comment]
Implemented in PR #553, closing.
